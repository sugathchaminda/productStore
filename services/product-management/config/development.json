{
    "version": "1.0.0",
    "api_version_prefix": "/api/v1",
    "port": 3000,
    "cors_urls": "*",
    "DB_HOST": "localhost",
    "DB_PORT": "3306",
    "DB_NAME": "PRODUCTMANAGER",
    "DB_USER": "root",
    "DB_PASSWORD": "Psalm54v2#",
    "DB_DIALECT": "mysql",
    "database": {
        "mongodb": {
            "host": "localhost",
            "port": 27017,
            "name": "xxxxxxx"
        }
      },
      "sql": {
        "db": "PRODUCTMANAGER",
        "user": "users_service",
        "password": "123"
      },
    "messageBus": "amqp://guest:guest@localhost:15672",
    "logs": {
        "folder": "./logs",
        "cron": "./logs/cron.log",
        "controller": "./logs/controller.log"
    },
    "sendgrid": {
        "api_key": "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"
    },
    "emails": {
        "fromName": "Xyz forum",
        "defaultFrom": "sugathsathkorala@gmail.com"
    },
    "jwt": {
        "jwt_common_secret": "ULYOHJE7MYWPHMHPVDFQAL9RRZ7BA4BN6F24RRQNHTNG8I08PY8F339FQ43R42UA",
        "jwt_auth_secret": "AA48973B1A76A4710DD3D49F14F2CFEFC85C779F04F6CE79B7980B06178A9D5E",
        "jwt_expire": "1h"
    }
  }