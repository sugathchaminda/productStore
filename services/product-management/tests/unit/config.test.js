/* eslint-disable no-undef */
import config from '../../config/config';

describe('config', () => {
  test('should load default values', () => {
    expect(config.port).not.toBeNull();
    expect(config).toMatchSnapshot();
  });
});
