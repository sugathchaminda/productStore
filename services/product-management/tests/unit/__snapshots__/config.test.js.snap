// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config should load default values 1`] = `
Object {
  "DB_DIALECT": "mysql",
  "DB_HOST": "localhost",
  "DB_NAME": "PRODUCTMANAGER",
  "DB_PASSWORD": "Psalm54v2#",
  "DB_USER": "root",
  "api_version_prefix": "/api/v1",
  "cors_urls": "*",
  "emails": Object {
    "defaultFrom": "sugathsathkorala@gmail.com",
    "fromName": "Xyz forum",
  },
  "jwt": Object {
    "jwt_auth_secret": "AA48973B1A76A4710DD3D49F14F2CFEFC85C779F04F6CE79B7980B06178A9D5E",
    "jwt_common_secret": "ULYOHJE7MYWPHMHPVDFQAL9RRZ7BA4BN6F24RRQNHTNG8I08PY8F339FQ43R42UA",
    "jwt_expire": "1h",
  },
  "logs": Object {
    "controller": "./logs/controller.log",
    "cron": "./logs/cron.log",
    "folder": "./logs",
  },
  "messageBus": "amqp://guest:guest@localhost:15672",
  "port": 3000,
  "sendgrid": Object {
    "api_key": "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",
  },
  "startedMessage": "undefined is running on port 3000/",
  "version": "1.0.0",
}
`;
